examples/features/broker-connection/amqp-sending-overssl/store-generation.txt

```diff
@@ -1,38 +1,55 @@
 # The various SSL stores and certificates were created with the following commands:
 # This can be run as a script by sourcing the file, e.g ". store-generation.txt"
+# Requires use of JDK 8+ keytool command.
+set -e
+
+KEY_PASS=securepass
+STORE_PASS=securepass
+CA_VALIDITY=365000
+VALIDITY=36500
+SERVER_NAMES="san=dns:localhost,dns:localhost.localdomain,dns:artemis.localtest.me,ip:127.0.0.1"
```

Comment:
```suggestion
SERVER_NAMES="san=dns:localhost"
```
The others aren't used so I think they should be removed and just match what the example does.

(squash the commit before you push if using this)